bench.py

```diff
@@ -2,12 +2,45 @@
 import os
 import sys
 import time
+import psutil
```

Comment:
I think it would be cleaner to have all codespeed upload functionality in another module, and just call it from this module.